### ISSUE TITLE
Multus reading stored conf file improperly after the node reboot.

### ISSUE THREAD

**Issue by ahalim-intel** [Intel-Corp/multus-cni#25]
After the next reboot we get the error. There is no file in /var/lib/cni directory. The kubelet logs however tell something weird:
```
Nov 08 03:20:30 Minion2 kubelet[1333]: W1108 03:20:30.421959    1333 docker_sandbox.go:342] failed to read pod IP from plugin/docker: NetworkPlugin cni failed on the status hook for pod "busybox-flannel-with-multus-minion2_default": CNI failed to retrieve network namespace path: Cannot find network namespace for the terminated container "1dbac58fcb0b562c3e1ef109c045c8599838646415d9b80f9dbc3f79eefd757d"
Nov 08 03:20:30 Minion2 kubelet[1333]: W1108 03:20:30.423128    1333 docker_sandbox.go:342] failed to read pod IP from plugin/docker: NetworkPlugin cni failed on the status hook for pod "busybox-flannel-with-multus-minion2_default": CNI failed to retrieve network namespace path: Cannot find network namespace for the terminated container "fc2572c319de6a10cda6f097e41b4d080c674e7556b4ec797f7a2aaf9160dc4b"
...
Nov 08 03:20:31 Minion2 kubelet[1333]: E1108 03:20:31.044895    1333 cni.go:312] Error deleting network: Multus: Err in  reading the delegates: failed to read container data in the path("/var/lib/cni/multus/fc2572c319de6a10cda6f097e41b4d080c674e7556b4ec797f7a2aaf9160dc4b"): open /var/lib/cni/multus/fc2572c319de6a10cda6f097e41b4d080c674e7556b4ec797f7a2aaf9160dc4b: no such file or directory

Nov 08 03:20:31 Minion2 kubelet[1333]: E1108 03:20:31.045660    1333 remote_runtime.go:114] StopPodSandbox "fc2572c319de6a10cda6f097e41b4d080c674e7556b4ec797f7a2aaf9160dc4b" from runtime service failed: rpc error: code = 2 desc = NetworkPlugin cni failed to teardown pod "busybox-flannel-with-multus-minion2_default" network: Multus: Err in  reading the delegates: failed to read container data in the path("/var/lib/cni/multus/fc2572c319de6a10cda6f097e41b4d080c674e7556b4ec797f7a2aaf9160Nov 08 03:20:31 Minion2 kubelet[1333]: E1108 03:20:31.045694    1333 kuberuntime_gc.go:154] Failed to stop sandbox "fc2572c319de6a10cda6f097e41b4d080c674e7556b4ec797f7a2aaf9160dc4b" before removing: rpc error: code = 2 desc = NetworkPlugin cni failed to teardown pod "busybox-flannel-with-multus-minion2_default" network: Multus: Err in  reading the delegates: failed to read container data in the path("/var/lib/cni/multus/fc2572c319de6a10cda6f097e41b4d080c674e7556b4ec797f7a2aaf9160dc4b
```

**Comment by rkamudhan** [comment]
@ahalim-intel Thanks. I will take up the bug, It should be fixed by PR #8

**Comment by rkamudhan** [comment]
@ahalim-intel bug fix is merged
